pkg/awsutils/awsutils.go

```diff
@@ -698,6 +709,33 @@ func (cache *EC2InstanceMetadataCache) tagENI(eniID string) {
 	})
 }
 
+//parseAdditionalEniTagsMap will create a map for additional tags
+func parseAdditionalEniTagsMap(additionalEniTags string) (map[string]string, error) {
+	var additionalEniTagsMap map[string]string
+
+	// If duplicate keys exist, the value of the key will be the value of latter key.
+	err := json.Unmarshal([]byte(additionalEniTags), &additionalEniTagsMap)
+	if err != nil {
+		log.Errorf("Invalid format for ADDITIONAL_ENI_TAGS. Expected a json hash: %v", err)
+	}
```

Comment:
I think it would be a good idea to add a blacklist of tags that are not allowed to be changed. For example:
* `cluster.k8s.amazonaws.com/name`
* `node.k8s.amazonaws.com/instance_id`
* Not yet, but soon `node.k8s.amazonaws.com/no_manage`

Any others we might want to disallow?

Reply:
I like the idea of disallowing anything with `k8s.amazonaws.com` prefix.